Kconfig

```diff
@@ -2230,6 +2230,38 @@ config DEBUG_PCI_INFO
 		Enable PCI driver informational output to SYSLOG.
 
 endif # DEBUG_PCI
+
+config DEBUG_RPTUN
```

Comment:
How about change `DEBUG_RPTUN` to `DEBUG_RPMSG` and also `rptunxxx()` --> `rpmsgxxx()`, I think all the rpmsg transport need used this debug option in the future.

Reply:
Will RPTUN and RPMSG co-exist in the future? if so we can add RPMSG related macros sepearately.

Reply:
RPTUN will depend on RPMSG, so it's better to add rpmsgxxx macro, and map rptunxxx to rpmsgxxx.